__tests__/plugin.test.js

```diff
@@ -75,7 +75,7 @@ describe('babel plugin', () => {
     `;
 
     const { code } = runPlugin(input);
-    expect(code).not.toContain('__reanimatedWorkletInit');
+    expect(code).not.toContain('_f.__worklet = true;');
```

Comment:
Do we need this `__worklet` prop? Looks like it only makes all the worklets generated code one line longer for no purpose. If we use it somewhere we should perhaps just check `__workletHash` prop that at least have some additional purpose

Reply:
Done in af8cda4443cce737d9d08ca7909626ccbb32be12